packages/test-utilities/package.json

```diff
@@ -2,20 +2,15 @@
   "name": "@fluentui/test-utilities",
   "version": "8.0.0-beta.7",
   "description": "Utilities used when testing components.",
-  "main": "lib-commonjs/index.js",
-  "module": "lib/index.js",
+  "main": "lib/index.js",
```

Comment:
Why was this changed from referencing a commonjs module to an es module?

Reply:
It shouldn't be, I changed this package to build commonjs only since it doesn't need ES modules.

Reply:
That's what `task('build', 'build:node-lib')` does. If that for some reason made it build ESM only I'll fix it.